.github/workflows/ruby.yml

```diff
@@ -8,15 +8,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: [1.8.x, 1.9.x, 2.0.x, 2.1.x, 2.2.x,
-          2.3.x, 2.4.x, 2.5.x, 2.6.x, 2.7.x, 3.0.x]
+        ruby: [2.3, 2.4, 2.5, 2.6, 2.7, 3.0]
 
     steps:
-    - uses: actions/checkout@v1
-    - name: Set up Ruby 2.6
-      uses: actions/setup-ruby@v1
+    - uses: actions/checkout@v2
+    - name: Set up Ruby ${{ matrix.version }}
```

Comment:
I think this line causes the checks to error.

Reply:
Yes you are right, fixed
![image](https://user-images.githubusercontent.com/2664467/116043352-3aef9c00-a678-11eb-969e-48e8ea715d25.png)